spec/rack/sinatra_spec.rb

```diff
@@ -26,4 +26,19 @@ def app
     get('/boom')
     expect(backend.reported_exceptions).to include(an_instance_of(CustomError))
   end
+
+  it 'populates request data into rollbar' do
```

Comment:
yeah… i didn't know the best place to put this. no strong opinions here, you think it's worth copying the sinatra config into the rollbar spec to test this? or is there an easier way?